common/djangoapps/student/models.py

```diff
@@ -1624,11 +1625,23 @@ def refund_cutoff_date(self):
         order_number = attribute.value
         try:
             order = ecommerce_api_client(self.user).orders(order_number).get()
+
         except HttpClientError:
             log.warning(
                 u"Encountered HttpClientError while getting order details from ecommerce. "
                 u"Order={number} and user {user}".format(number=order_number, user=self.user.id))
+            return None
+
+        except HttpServerError:
+            log.warning(
+                u"Encountered HttpServerError while getting order details from ecommerce. "
+                u"Order={number} and user {user}".format(number=order_number, user=self.user.id))
+            return None
 
+        except SlumberBaseException:
```

Comment:
@schenedx I have added `SlumberBaseException` if there is any new exception occur in future will be handled here.